src/py42/exceptions.py

```diff
@@ -107,6 +109,15 @@ class Py42InternalServerError(Py42HTTPError):
     """A wrapper to represent an HTTP 500 error."""
 
 
+class Py42ActiveLegalHoldError(Py42BadRequestError):
+    """An exception raised when attempting to deactivate a user or device that is in an
+    active legal hold."""
+
+    def __init__(self, exception):
+        msg = u"Cannot deactivate from user in an active legal hold."
```

Comment:
I suspect we may want to update this message.

Reply:
Could add a `resource` parameter that is either `user` or `device`, along with an `uid` parameter that is either the `device_guid` or the `userUid` and the message could be like `u"Cannot deactivate the {0} with UID {1} as the {0} is involed in a legal hold matter.".format(resource, uid)`.